requirements/base-requirements.in

```
@@ -67,8 +67,11 @@ jsonobject
 jsonpath-ng @ https://github.com/kaapstorm/python-jsonpath-rw/raw/wherenot+find_or_create/wheel/jsonpath_ng-1.5.2.2-py3-none-any.whl
 jsonschema
 kafka-python
+# missing celery specified as parent dependency in .txt files. Check https://github.com/jazzband/pip-tools/issues/1505
 kombu
```

While I know this goes with `kombu`, it wasn't immediately obvious because I'm used to `pip-tools` convention of placing dependent comments indented below the dependency. What do you think of this?
```suggestion
kombu
    # via celery
    # See https://github.com/jazzband/pip-tools/issues/1505
    # This comment can be removed when that issue is resolved.
```

---

Good suggestion. See 4f4d4a5025b4f630337640c6221fa2ae19295d11

---

:+1: 

Guess you decided not to indent the comments? (not a big deal)

---

I gotchu